libs/openFrameworks/graphics/ofImage.cpp

```diff
@@ -222,7 +222,7 @@ static bool loadImage(ofPixels_<PixelType> & pix, const of::filesystem::path & _
 	uriFreeUriMembersA(&uri);
 
 	if(scheme == "http" || scheme == "https"){
-		return ofLoadImage(pix, ofLoadURL(_fileName.string()).data);
+		return ofLoadImage(pix, ofLoadURL(ofPathToString(_fileName)).data);
```

Comment:
@dimitre - so this works? you can pass a path that starts https::// and it won't error?

Reply:
I think this is already being done via of::filesystem::path in master, the unique difference is this new function has the try / catch.
I suppose the tests ```tests/io/loadImage``` are testing this situation and passing with https